workinggroups/ci.md

```diff
@@ -0,0 +1,34 @@
+# CNCF CI WG Proposal
```

Comment:
Could a template file be created with the sections that are wanted and comments (using HTML comments) to explain how to fill them in? It could be called `TEMPLATE.md` and everyone could use that as a starting point. It would lead them to fill in the details the way they are needed.

Reply:
I wrote up an idea for a template... https://gist.githubusercontent.com/mattfarina/5738bab306e3907c5c523156dbc21469/raw/6c6b9bfc42451ed2a78f3448e3632b250ab93e86/TEMPLATE.md

Reply:
I like the template!

Reply:
If this is a real proposal for a CI WG, then I think it should be split into separate PR.